docs/how-to-run.md

```diff
@@ -0,0 +1,168 @@
+```bash
+$ docker-compose -f docker-compose.yaml -f docker-compose-rabbitmq.yml up -d --force-recreate bblfsh postgres rabbitmq
+```
+
+To monitor RabbitMQ, go to http://localhost:8081, and access it with `guest/guest`
+
+# Run the Analyzers
+
+You will need to run the Analyzers to be used by lookout.
+
+You can run one of our [example analyzers](analyzers-examples.md), any of the already [available analyzers](../README.md#available-analyzers) or the one that you're developing.
+
+For testing purposes, you may want to use a *dummy* analyzer. You can do so downloading it from [lookout releases page](https://github.com/src-d/lookout/releases) and running:
+
+```bash
+$ dummy serve
+```
+
+# Configure lookout
+
+Copy the [`config.yml.tpl`](/config.yml.tpl) into `config.yml` and modify it according to your needs.
+
+Take a look at [configuration and GitHub authentication](configuration.md) for more details about **lookout** configuration.
+
+At least you should:
+
+1. add the gRPC addresses of the analyzers you ran in the previous step,
+1. add the URLs of the repositories to be watched or authenticate as a GitHub App.
+
+
+# Run lookout
+
+Download the latest `lookoutd` binary from [lookout releases page](https://github.com/src-d/lookout/releases).
+
+## Initialize the Database
+
+_For non-default configuration, please take a look into [**`lookoutd` Command Options**](#options)_
+
+```bash
+$ lookoutd migrate
+```
+
+## Start lookout
+
+_For non-default configuration, please take a look into [**`lookoutd` Command Options**](#options)_
+
+For a single server watching GitHub and processing events, just run:
+
+```bash
+$ lookoutd serve [--dry-run] [--github-token=<token> --github-user=<user>]
+```
+
+### Distributed Mode
+
+_For non-default configuration, please take a look into [**`lookoutd` Command Options**](#options)_
+
+In order to run it in a distributed mode, the _watcher_ and the _workers_ must be run separately.
+
+Run the _watcher_:
+
+```bash
+$ lookoutd watch [--github-token=<token> --github-user=<user>]
+```
+
+and as many _workers_ you need:
+
+```bash
+$ lookoutd work [--dry-run] [--github-token=<token> --github-user=<user>]
+```
+
+<a id=options></a>
+# Appendix: `lookoutd` Command Options
+
+`lookoutd` binary include some subcommands as described above, and they accept many different options; you can use:
+- `lookoutd -h`, to see all the available subcommands.
+- `lookoutd subcommand -h`, to see all the options for the given subcommand.
+
+Here are some of the most relevant options for `lookoutd`:
+
+- [dry-run mode](#dry-run-mode),
+- [authentication options](#authentication-options),
+- [number of concurrent events to process](#concurrent-events),
+- [dependencies URIs](#dependencies-uris)
+- [logging options](#logging-options)
+
+## Dry-run Mode
+If you want to avoid posting the analysis results on GitHub, and only print them, enable the _dry-run_ mode when running `serve`, `work` subcommands:
+
+| subcommands | Env var | Option |
```

Comment:
looks like tables are broken:
https://github.com/src-d/lookout/blob/fdd5d732139120ce7e87f2e370de9ec6c55fbd0f/docs/how-to-run.md

Reply:
whaaat? how did I ruin them!!?
thanks for spotting that problem!

Reply:
fixed by 3912bed

Reply:
fixed by e3d47eb